src/Features/Core/Portable/FindUsages/IRemoteFindUsagesService.cs

```diff
@@ -255,22 +281,26 @@ public async ValueTask<DefinitionItem.DefaultDefinitionItem> RehydrateAsync(Solu
     public readonly SerializableDocumentSpan SourceSpan = sourceSpan;
 
     [DataMember(Order = 2)]
-    public readonly SymbolUsageInfo SymbolUsageInfo = symbolUsageInfo;
+    public readonly SerializableClassifiedSpansAndHighlightSpan ClassifiedSpans = classifiedSpans;
 
     [DataMember(Order = 3)]
+    public readonly SymbolUsageInfo SymbolUsageInfo = symbolUsageInfo;
+
+    [DataMember(Order = 4)]
     public readonly ImmutableDictionary<string, string> AdditionalProperties = additionalProperties;
 
     public static SerializableSourceReferenceItem Dehydrate(int definitionId, SourceReferenceItem item)
         => new(definitionId,
                SerializableDocumentSpan.Dehydrate(item.SourceSpan),
+               // We're always have classified spans for C#/VB, which are the only languages used in OOP find-references.
+               SerializableClassifiedSpansAndHighlightSpan.Dehydrate(item.ClassifiedSpans!.Value),
```

Comment:
the computation always happens here: https://github.com/dotnet/roslyn/blob/cdbed67ceb23729c0df2bbc4c06d93b8d5d72e2d/src/Features/Core/Portable/FindUsages/DefinitionItemFactory.cs#L303-L310

we just were dropping the value on the floor here.